torch/_inductor/kernel/flex_attention.py

```diff
@@ -201,6 +232,27 @@ def build_subgraph_buffer(
     q_offset = off_z * stride_qz + off_h * stride_qh
     k_offset = off_z * stride_kz + off_h * stride_kh
     v_offset = off_z * stride_vz + off_h * stride_vh
+
+    SPARSE_Z = {{size("SPARSE_KV_NUM_BLKS", 0)}}
```

Comment:
can you update the forward and backward kernel with the args and constexprs descriptions

Reply:
done